src/components/App.svelte

```diff
@@ -1,7 +1,31 @@
 <script>
-  export let name = 'Svelte'
+  import Header from './Header';
+  import Slider from './Slider';
+  import TestSlide from './TestSlide';
+  import Footer from './Footer';
 </script>
 
-<h1 class="text-red-500 text-5xl text-center font-hairline py-8">
-  Hello, it's me, {name}!
-</h1>
+<style>
+
+</style>
+
+<div class="container mx-auto p-3 sm:py-4">
+  <Header />
+</div>
+<main>
+  <div class="lg:hidden py-8">
+    <Slider>
+      {#each [1, 2, 3] as number (number)}
```

Comment:
On preview it opens a second slide by default. Is it by design or it's a bug?

Reply:
It's by design, activeSlide === 1 by default

Reply:
It's a bit strange.
When there's a list, I would expect the first item from the beginning

Reply:
It was just for a test. I wanted immediately see that slide is on the screen center and I can see slides on the sides)

Reply:
For this purpose, I think storybook has some functionality, but not sure about svelte. With React has for sure